core/backend-itwin-client/src/backend-itwin-client.ts

```diff
@@ -10,7 +10,6 @@ export * from "./imodelhub/LocalhostFileHandler";
 export * from "./imodelhub/AzureFileHandler";

```

Comment:
Looks like you remove the new file entirely, can you add it back here?

Reply:
done